single-line-diagram/single-line-diagram-cgmes/single-line-diagram-cgmes-dl-conversion/src/main/java/com/powsybl/sld/cgmes/dl/conversion/AbstractDiagramDataExporter.java

```diff
@@ -39,7 +39,7 @@ protected AbstractDiagramDataExporter(TripleStore tripleStore, ExportContext con
     }
 
     protected String addDiagramObject(String id, String name, double rotation, String diagramObjectStyleId, String diagramId) {
-        PropertyBag diagramObjectProperties = new PropertyBag(Arrays.asList(CgmesDLModel.IDENTIFIED_OBJECT_NAME, CgmesDLModel.IDENTIFIED_OBJECT, "rotation", CgmesDLModel.DIAGRAM, CgmesDLModel.DIAGRAM_OBJECT_STYLE));
+        PropertyBag diagramObjectProperties = new PropertyBag(Arrays.asList(CgmesDLModel.IDENTIFIED_OBJECT_NAME, CgmesDLModel.IDENTIFIED_OBJECT, "rotation", CgmesDLModel.DIAGRAM, CgmesDLModel.DIAGRAM_OBJECT_STYLE), true);
```

Comment:
Do we need to decode identifiers?

Reply:
I don't know which option is the most suitable. It could be a layout parameter with a default value to true or false instead of a hard-coded value.